apps/web/components/v2/apps/App.tsx

```diff
@@ -0,0 +1,337 @@
+import classNames from "classnames";
```

Comment:
Any logic in this file (for eg pricing, showing of buy button etc) is coming directly from v1. I only touched styling in this file. 

Only new thing is that `<Slider>` isn't shown directly in the body anymore. We now extract the images from the MDX's meta data, and render it in its own column.